main.py

```diff
@@ -0,0 +1,74 @@
+import tkinter as tk
+from tkinter import ttk
+from tkinter import filedialog
+import matplotlib.pyplot as plt
+import json
+
+# Set color values
+LIGHT_BLUE = '#e3f8fb'
+
+
+# Matplotlib logic
+class PlotView:
+    """A class that handles parsing the JSON file and visualizing the graph"""
+    def __init__(self, file_name):
+        """Getting the file path, opening it and calling a method to load the information"""
+        self.file_name = file_name
+        self.json_f = open(file_name)
+        self.load_info()
+
+    def load_info(self):
+        """Loading data from the file, parsing it to according variables, calling а method to visualize the plot"""
```

Comment:
It's a good practice to have the docstrings in imperative form like so: "Load data from the file, parse it to according variables, call a method to visualize the plot"